docs/dev/developer.rst

```diff
@@ -10,9 +10,72 @@ RADIS is an open-source project, and therefore anyone can contribute, whether yo
 know spectroscopy or not. The project is organized around a
 `GitHub repository <https://github.com/radis/radis/>`__.
 
-You want to become a `contributor <https://github.com/radis/radis/graphs/contributors>`__? Welcome!
-First follow the install procedure for developpers below. Then, you can have a look at all the `GitHub opened issues <https://github.com/radis/radis/issues>`__.
-and tackle specifically one of the `Good First Issues <https://github.com/radis/radis/contribute>`__.
+Getting Started
+=============
+
+New to open source? Welcome! 👋 Here's how to get started:
+
+1. Look for `Good First Issues <https://github.com/radis/radis/contribute>`__ - these are specifically tagged to help new contributors.
+2. The `Documentation TODO List <https://github.com/radis/radis/issues/77>`__ is also a great place to start.
+3. Feel free to ask questions in our community chats:
+
+.. image:: https://badges.gitter.im/Join%20Chat.svg
+    :target: https://gitter.im/radis-radiation/community
+    :alt: Gitter
+
+.. image:: https://img.shields.io/badge/slack-join-green.svg?logo=slack
+    :target: https://radis.github.io/slack-invite/
+    :alt: Slack
+
+Making Changes
+============
+
+We use the `GitHub Flow <https://guides.github.com/introduction/flow/index.html>`__ for all code changes:
+
+1. Fork the repository
+2. Create a branch for your feature
+3. Make your changes
+4. Open a Pull Request (PR)
+5. Address review comments
+6. Get merged!
+
+For small changes (like fixing typos), you can edit directly on GitHub and open a PR.
+
+Code Style and Linting
+--------------------
```

Comment:
Before, this information was before in a file called `_linting.rst`. I think it's fine with your commit but please remove any suggestion of this deprecated file.